src/lib.rs

```diff
@@ -124,8 +124,6 @@ extern crate std;
 #[cfg(not(feature = "std"))]
 extern crate core as std;
 
-#[cfg(feature = "blas")]
-extern crate blas_src;
```

Comment:
This was the main feature in the old solution - when we don't link this, the end user has to.